pkg/pushsync/pushsync.go

```diff
@@ -480,9 +484,18 @@ func (ps *PushSync) pushToNeighbour(peer swarm.Address, ch swarm.Chunk, origin b
 	// price for neighborhood replication
 	receiptPrice := ps.pricer.PeerPrice(peer, ch.Address())
 
+	// decouple the span data from the original context so it doesn't get
+	// cancelled, then glue the stuff on the new context
+	span := tracing.FromContext(ctx)
+
 	ctx, cancel := context.WithTimeout(context.Background(), timeToWaitForPushsyncToNeighbor)
```

Comment:
Since ctx is now passed to pushToNeighbour, could here context.Background() be replaced with ctx?

Reply:
I'm passing it just so I can extract the OT span info out of it. The reason we're using context.Background is so that the parent context doesn't expire while we do the replication